climpred/prediction.py

```diff
@@ -23,7 +22,7 @@
 # predictability.
 # --------------------------------------------#
 @is_xarray([0, 1])
-def compute_perfect_model(ds, control, metric='rmse', comparison='m2e'):
+def compute_perfect_model(ds, control, metric='pearson_r', comparison='m2e'):
```

Comment:
Curious why pearson_r preferred over rmse now. Actually maybe metric shouldn't be optional?

Reply:
For consistency with `compute_hindcast`. I figure computing an ACC is the most common metric to run. But perhaps you're right that we should make it mandatory to input the metric so the user is very aware of what they're computing. Could be done in another PR. I'll leave this for now.